mdc_workstation_cards_employee_rel/hooks.py

```diff
@@ -0,0 +1,19 @@
+from odoo import _, api, SUPERUSER_ID
+from datetime import datetime, timedelta
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        current_employees = env['mdc.workstation'].search([
+            ('current_employee_id', '!=', False)])
+        for reg in current_employees:
+            # Modify workstation cards with the employee in this workstation
+            workstation_cards = env['mdc.card'].search(
+                [('card_categ_id', '=', env.ref('mdc.mdc_card_categ_L').id)
+                    , ('workstation_id', '=', reg.id)])
+            if workstation_cards:
+                for card in workstation_cards:
+                    card.write({
+                        'employee_id': reg.current_employee_id.id
+                    })
```

Comment:
This also should work (https://www.odoo.com/documentation/11.0/reference/orm.html#odoo.models.Model.write):
```suggestion
                workstation_cards.write({
                    'employee_id': reg.current_employee_id.id
                })
```

Reply:
Yes, I know it. But in this case, in this model, the write method has a self.ensure_one(). Then that improvement does not work. 
https://github.com/solvosci/manufacture/blob/439c0843f25dbea63d6cddf7e7a0f9aa2842c4f4/mdc/models/structure.py#L253

Reply:
Oopps I didn't see it